packages/framework/ai-collab/src/test/ai-collab/jobBoardBenchmark.spec.ts

```diff
@@ -342,6 +343,7 @@ describe.skip("AI Job Listings App Benchmark", () => {
 		}
 		const foundJohnDoe = createJohnDoeCandidateTask.data;
 		assert(foundJohnDoe !== undefined);
+		assert(response.diffs !== undefined);
```

Comment:
@chentong7 

Shouldn't we `unskip` the whole test suite (it's currently skipped in line `223`)

Reply:
`Error: 401 You didn't provide an API key. You need to provide your API key in an Authorization header using Bearer auth (i.e. Authorization: Bearer YOUR_KEY), or as the password field (with blank username) if you're accessing the API from your browser and are prompted for a username and password. You can obtain an API key from https://platform.openai.com/account/api-keys`

This is what I get when I run the test. I'd like to spend some time in tmrw's parking lot to ramp up on some AI-collab testing library / concepts 🤯 .

Reply:
Reading through the suite, and the fact that it requires an API_KEY, this test is not a unit test, is more of an integration/e2e test. The code lives here now because here's where everything started, but it needs to move to a different location (potentially the e2e tests package?) later. So it should remain skipped as long as it's here.

Reply:
If we can extract the mapping of EditLog to Diff to a separate function we can use unit tests instead of this integration test. It isn't quite deterministic given we have set the temperature value on the model, and even when its set to be the most deterministic, technically it can give a different response.